src/i18n/locales/index.ts

```diff
@@ -0,0 +1,29 @@
+import translationEN from './en';
+import translationPSEUDO from './pseudo';
+
+export enum i18nLanguageCode {
+  ENGLISH = 'en',
+  PSEUDO = 'pseudo'
+}
+
+export const defaultResources = {
+  [i18nLanguageCode.ENGLISH]: {
+    name: 'English',
+    ...translationEN
+  }
+};
+
+export const resources = {
+  [i18nLanguageCode.ENGLISH]: {
+    name: 'English',
+    ...translationEN
+  },
+  ...(process.env.NODE_ENV === 'development'
+    ? {
+        [i18nLanguageCode.PSEUDO]: {
+          name: 'Pseudo',
+          ...translationPSEUDO
+        }
+      }
+    : {})
```

Comment:
I was originally thinking of using an `if (process.env.NODE_ENV === 'development')` and setting the property value, but this actually looks cleaner.

Destructuring like this would also allow only enabling languages that are "in beta" (e.g. very incomplete/in-progress translation maps) to be turned on in development only. Which makes me think: Perhaps we can take this out into a constant `developmentOnlyLanguages`?